src/test/java/net/snowflake/client/core/SessionUtilLatestIT.java

```diff
@@ -465,4 +465,56 @@ public void testOktaAuthRetry() throws Throwable {
       SessionUtil.openSession(loginInput, connectionPropertiesMap, "ALL");
     }
   }
+
+  @Test
+  public void testOktaDisableSamlUrlCheck() throws Throwable {
```

Comment:
let's add a javadoc to this method that this latest test will work with jdbc > 3.16.0

Reply:
Added.